workflows/crisprseq_screening.nf

```diff
@@ -162,9 +165,12 @@ workflow CRISPRSEQ_SCREENING {
             counts
         )
 
+        ch_versions = ch_versions.mix(MAGECK_TEST.out.versions)
+
         MAGECK_GRAPHRRA (
             MAGECK_TEST.out.gene_summary
         )
+        //TODO: Missing versions
```

Comment:
Why are they missing?

Reply:
well at least my todo to not forget it worked... thanks for spotting it :D